### PR TITLE
[lab-prob-distributions]Alejandro Aboy

### PR DESCRIPTION
**Pull request by aboyalejandro**

### CODE REVIEW

**Comment by ta-data-pt-rmt** [comment]
Lab checked Alejandro,

Great work, keep it going!Most exercises were on point 🔥 🎯 

**Quick notes:**

**-Continuous:**
For the normal distribution histograms, the two distributions differ in their range of values (one is just positive and other range for (negative, positive). We can also say that the first has higher kurtosis (higher spike).

In order to access and evaluate the histograms, it's a good practice to increase the number of bins so that we increase angularity on the plot, and by result, be able to better analyse which plot looks more like a Normal Distribution 😄 


**-Discrete:**

It is normal that the first questions may have caused confusion since we didn't really talked about it, and are not the main point of concern. However here are the solutions:

-What is the probability that the first 5 fruits are all apples?
apple5 = p** 5 == 0.6 ** 5  (because we have replacement)

-What is the probability that the first 5 fruits are all apples and the next 15 fruits are all oranges?
apple5_orange15 = apple5 * (0.4**15)

-What is the probability that this time the sample contains 5 apples and 15 oranges?
You commented, very well, that if both probs should be aggregated or not  and no, you don't need to aggregate since we only have 2 types of fruits, so the condition of 1 affect the other fruit. 


-What is the probability that less than 5 fruits picked are apples?
binomial_dist1.cdf(**4**) -->statistics love to play word games a lot, so when you see just less you don't include the actual number


-What is the probability that the number of goals scored in a match is 5?
-Hint: Use the exp() in the math library:

Basically the exercise wanted you to use the formula of the poisson distribution, which is not mandatory to know by default and thank statitian jesus we have scikit learn for that.

But just a reminder that factorial (also known as !) is in example:     5! = 5x4x3x2x1 
x = 5
mu = 2.3
prob = (math.exp(-2.3)*mu**x)/math.factorial(x)


All other exercises were on point! Good of you for outlining what you didn't understand  🎯 👊
